### PR TITLE
Add log to fifottl

### PR DESCRIPTION
**Pull request by Alex-Kuz**

### CODE REVIEW

File: sharded_queue/drivers/fifottl.lua [hunk 1]
```diff
@@ -259,6 +259,9 @@ function method.put(args)
         }
     end)
 
+    log.info(("[%d] put %d"):format(
```
Comment (akudiyar) [inline sharded_queue/drivers/fifottl.lua:262]: [%d] put task %d

Comment (akudiyar) [inline sharded_queue/drivers/fifottl.lua:262]: Add `args.options.log_requests = true|false` and a guard for optional logging (at each call).

File: sharded_queue/api.lua [hunk 2]
```diff
@@ -77,8 +94,13 @@ function sharded_tube.take(self, timeout, options)
     if options == nil then
         options = {}
     end
+    
     options.tube_name = self.tube_name
 
+    if options and options.log_request == nil then
```
Comment (akudiyar) [inline sharded_queue/api.lua:100]: `options` is always not nil

File: sharded_queue/api.lua [hunk 3]
```diff
@@ -40,11 +49,19 @@ function sharded_tube.put(self, data, options)
     options.tube_name = self.tube_name
     options.bucket_id = bucket_id
     options.bucket_count = bucket_count
+    options.options = {
+        log_request = utils.normalize.log_request(options.log_request)
```
Comment (akudiyar) [inline sharded_queue/api.lua:53]: Seems that normalize is useless here (we do not validate other options)

Comment (Alex-Kuz) [reply sharded_queue/api.lua:53]: I think, there is a substitution of concepts: in fact, the parameters above are not optional, but mandatory, their invalidity leads to an error. And the `log_request` is an optional parameter, and it's invalidity does not lead to errors, however, we should ignore the user's attempts to pass a string or number 42 into this parameter.
Therefore, I think `normalize` is necessary.

File: sharded_queue/api.lua (hunk 2 above)
Comment (Alex-Kuz) [reply sharded_queue/api.lua]: Fix here https://github.com/tarantool/sharded-queue/pull/31/commits/1c83c6676da9cfbbe1abaa66b369bdfd26d12922

**Comment by Alex-Kuz** [comment]
How to use:
```lua
conn:call('queue.create_tube', { mytube, {
    log_request = true, -- log all operations
}})

conn:call("queue.tube.mytube:put", { data, {
    log_request = false, -- this PUT will not be logged
}})

conn:call("queue.tube.mytube:put", { anoter_data }) -- and this PUT will be logged
```

**Pull request merged**